apps/meteor/tests/e2e/e2e-encryption.spec.ts

```diff
@@ -654,6 +654,68 @@ test.describe.serial('e2ee room setup', () => {
 		await expect(poHomeChannel.content.lastUserMessage.locator('.rcx-icon--name-key')).toBeVisible();
 	});
 
+	test('expect save password state on encrypted DM', async ({ page }) => {
+		await page.goto('/account/security');
+		await poAccountProfile.securityE2EEncryptionSection.click();
+		await poAccountProfile.securityE2EEncryptionResetKeyButton.click();
+
+		await page.locator('role=button[name="Login"]').waitFor();
+
+		await page.reload();
+
+		await page.locator('role=button[name="Login"]').waitFor();
+
+		await injectInitialData();
+		await restoreState(page, Users.admin);
```

Comment:
I'm a little lost here, why the `waitFor` calls and the page reload? Won't restoreState login with the admin token?

Reply:
Page reload was due to an issue, where the websocket connection was dying on logout and wasn't working as expected on logging back again without a reload.

This issue was recently solved, I'll remove this page reload call.